app/tests/test_collections_model.py

```diff
@@ -0,0 +1,500 @@
+import logging
+from datetime import datetime
+
+from django.conf import settings
+from django.test import TestCase
+
+from stac_api.models import Item
+from stac_api.utils import utc_aware
+
+import tests.database as db
+
+logger = logging.getLogger(__name__)
+
+API_BASE = settings.API_BASE
+
+
+class CollectionsModelTestCase(TestCase):
```

Comment:
Very good unittest cover :+1:  just a general comment regarding unittest I would not use docstring as it clutter the unittest output IMHO:
```
test_updating_collection (tests.test_collections_endpoint.CollectionsEndpointTestCase) ... ok
test_update_temporal_extent_datetime (tests.test_collections_model.CollectionsModelTestCase)
Tests if the collection's temporal extent is correctly updated, when ... ok
test_update_temporal_extent_datetime_mixed_items (tests.test_collections_model.CollectionsModelTestCase)
Tests, if collection's temporal extent is updated correctly when ... ok
test_update_temporal_extent_deletion_datetime_item (tests.test_collections_model.CollectionsModelTestCase)
Two items with singel datetime values are added and one is deleted ... ok
test_update_temporal_extent_deletion_last_datetime_item (tests.test_collections_model.CollectionsModelTestCase)
An item is added to the collection and deleted again afterwards. ... ok
test_update_temporal_extent_deletion_last_range_item (tests.test_collections_model.CollectionsModelTestCase)
An item is added to the collection and deleted again afterwards. ... ok
test_update_temporal_extent_deletion_range_item (tests.test_collections_model.CollectionsModelTestCase)
Two items are added to the collection and one is deleted afterwards. ... ok
test_update_temporal_extent_range (tests.test_collections_model.CollectionsModelTestCase)
Tests if the collection's temporal extent is correctly updated, when ... ok
```
Uses at best regular comment which are not added to the unittest output

Reply:
Is it possible to configure the test runner differently? Cause I like the ocmments

Reply:
I don't think so, there isn't much to configure in the original test runner, or its not documented. Using regular comment `# ` instead of docstring works thouh

Reply:
replaced with regular comments now